site/docs/v1/tech/core-concepts/scim.adoc

```diff
@@ -170,6 +170,38 @@ In order to use FusionAuth as your SCIM provisioning server for SCIM clients, yo
 
 See the link:/docs/v1/tech/apis/scim/[SCIM API Overview] for details about the supported endpoints.
 
+== Client SCIM Authentication
+
+FusionAuth requires a completed Client Credentials grant and the resulting token for SCIM server authentication, as mentioned above. The token generated from the grant is used in the `Authorization` header as a `Bearer` token.
+
+However, some SCIM clients can't dynamically complete a client credentials grant. Instead, they need a static authorization header value.
+
+To correctly integrate such clients, do the following:
+
+* Change the JWT duration of the SCIM Client Entity Type to a larger value, such as a year, which is `31536000`.
+* Gather the SCIM Client Client Id, the SCIM Client Client Secret, and the SCIM Server Client Id. These values are available from the respective Entity details screen.
+* Determin the SCIM permissions you want to grant the SCIM Client.
+* Perform the Client Credentials grant manually using a tool like curl (see below for an example).
+
+When the token expires, the SCIM integration will fail. Document how to regenerate the token and update the SCIM client configuration internally.
+
+[source,shell,title="Performing the Client Credentials grant for a SCIM Client."]
```

Comment:
This is nice because you can copy and paste it, but I think many will find escaped values difficult to read. 

We could also show an example JSON body using a code snippet and then use the `@` file include syntax with `curl` to demonstrate the request.

Reply:
I reworked this.